Backend/updateBackendEnv.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
 pip install virtualenv
-virtualenv -p python2.6 env
```

Comment:
I've already mentioned this, I've talked with jon and this is the wanted approach. Prod will be wiped and built in stagings image. In our eyes prod isn't prod until its rebuilt.